bundles/org.eclipse.epsilon.emc.rdf.dt/src/org/eclipse/epsilon/emc/rdf/dt/RDFModelConfigurationDialog.java

```diff
@@ -515,16 +516,16 @@ public void modifyText(ModifyEvent event) {
 	}
 	
 	
-	protected String validateModel;
-
-	protected Button validateModelCheckBox;
+	protected Combo validateModelCombo;
 	private Composite createValidateModelGroup(Composite parent) {
 		final Composite groupContent = DialogUtil.createGroupContainer(parent, "Model validation", 1);
 
-		validateModelCheckBox = new Button(groupContent, SWT.CHECK);
-		validateModelCheckBox.setText("Run Jena's model validation on loaded models");
-		boolean isJenaValidationEnabled = RDFModel.VALIDATION_SELECTION_JENA.equals(validateModel);
-		validateModelCheckBox.setSelection(isJenaValidationEnabled);
+		validateModelCombo = new Combo(groupContent,SWT.READ_ONLY);	
+		validateModelCombo.setItems(RDFModel.VALIDATION_MODES);
+		
+		if(validateModelCombo.getText() == "") {
```

Comment:
Don't use == to compare strings: this doesn't work reliably. Use isBlank() here.